testcases/kernel/controllers/memcg/control/memcg_control_test.sh

```diff
@@ -116,7 +116,7 @@ result()
 cleanup()
 {
 	if [ -e $TST_PATH/mnt ]; then
-		umount $TST_PATH/mnt 2> /dev/null
+		umount -l $TST_PATH/mnt 2> /dev/null
 		rm -rf $TST_PATH/mnt
 	fi
 }
```

Comment:
I found that wait $! has no effect, because resource will be released after:
"echo 0 > mnt/memory.use_hierarchy 2> /dev/null"
If I sleep 1 second after this line, everything is OK; but if I sleep 5 seconds before this line, there is still erros.